x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authc/AuthenticationTests.java

```diff
@@ -209,38 +218,22 @@ public void testRunAsAccessResourceOf() {
     }
 
     public void testIsServiceAccount() {
```

Comment:
Service account does not support run-as nor run-by another user. So the old test is not really correct. It is fixed accordingly.